util/mini-tt2.pl

```diff
@@ -98,6 +98,16 @@
 my ($in_if, $in_else, $if_branch_hit);
 my ($in_block, $block, %blocks);
 my (%ctl_cmds, $prev_continuing_macro_line);
+my $modified = localtime((stat($infile))[9]);
```

Comment:
Too many parens there. We can save some when there is no operator precedence ambiguity.

Reply:
Actually we don't need this anymore since we are not outputting the modified time. Removed.